myapp/views.py

```diff
@@ -18,17 +24,23 @@ def index(request):
 def single_stock(request, symbol):
 	
 	data = stock_api.get_stock_info(symbol)
-	all_companies = stock_api.get_currency()
-	#check for currency type and add it to data
-	for object in all_companies:
-		if object['symbol'] == symbol:
-			currency = object['currency']
-			break
-
-	data['currency'] = currency
+	# all_companies = stock_api.get_currency()
+	# #check for currency type and add it to data
+	# for object in all_companies:
+	# 	if object['symbol'] == symbol:
+	# 		currency = object['currency']
+	# 		break
+    #
+	# data['currency'] = currency
 
 	return render(request, 'single_stock.html', {'page_title': 'Stock Page - %s' % symbol, 'data': data})
 
+def two_stocks(request, symbol1,symbol2):
```

Comment:
what does two_stocks do? try finding a better name for functions. long names are better then names other developers cant understand